ONSdigital/eq-author-app
# EAR-2076-adjust-max-digit-limit-for-validation

### What is the context of this PR?

This PR is linked to this [Jira ticket](https://jira.ons.gov.uk/browse/EAR-1998)

### How to review

- Create a question page with two number answers and:
  - enable total validation
  - check that the limit can be adjusted to a maximum of 15 digits

Check for currency and percentage answer types as well

### What to do after everything is green

1. - [ ] Bring this branch up-to-date with Origin/Master
2. - [ ] If there are a lot of commits or some are not helpful to read, squash them down
3. - [ ] Click the **Merge** button on this pull request
4. - [ ] Does this change mean the **Capability examples** questionnaire should be updated?
5. - [ ] Move the Jira ticket for this task into the next stage of the process
